src/app/settings/panel-settings/components/panel-configurations/panel-configurations.component.html

```diff
@@ -4,6 +4,11 @@
             <td class="td-width">{{panelConfiguration.type}} - {{panelConfiguration.model}}</td>
             <td class="td-width">{{panelConfiguration.hostname}}</td>
             <td>{{getPanelLayoutConfigurationNameForPanelConfiguration(panelConfiguration)}}</td>
+            <td class="text-right">
+                <sofie-button-group>
+                    <sofie-icon-button [icon]="Icon.TRASH_CAN" [iconSize]="IconSize.L" [tooltip]="deleteTooltipMessage" (click)="deletePanelConfiguration(panelConfiguration)"></sofie-icon-button>
```

Comment:
```suggestion
                    <sofie-icon-button [icon]="Icon.TRASH_CAN" [iconSize]="IconSize.L" [tooltip]="deleteTooltipMessage" (click)="deletePanelConfiguration(panelConfiguration)" />
```

Just a suggestion :)